TodoList/sources/TodoList.Services/Dependency/Bootstrapper.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.Practices.Unity;
+using TodoList.Contracts.Dependency;
+using TodoList.Contracts.Services;
+using TodoList.Services.Static_Wrappers;
+using TodoList.Services.Nodes;
+
+namespace TodoList.Services.Dependency
+{
+    public class Bootstrapper : IBootstrapper
+    {
+        public IUnityContainer RegisterType(IUnityContainer container)
+        {
+            container.RegisterType<IGenerateIdService, GenerateIdService>(new HierarchicalLifetimeManager());
```

Comment:
These stateless services could be singleton or transient, but scoped is an odd choice.